opds_import.py

```diff
@@ -1456,13 +1456,13 @@ def __init__(self, _db, collection, import_class,
                 )
             )
 
-        data_source = collection.data_source
+        data_source = self.data_source(collection)
```

Comment:
The 'Feedbooks' protocol has a hard-coded data source (Feedbooks) and, unlike other OPDS-type protocols, doesn't rely on a data source being configured on the collection.